### PR TITLE
Support standard labels for nodeexporter

### PR DESCRIPTION
**Pull request by rajatvig**

### CODE REVIEW

File: manifests/node-exporter-daemonset.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ kind: DaemonSet
 metadata:
   labels:
     app: node-exporter
```
Comment (brancz) [inline manifests/node-exporter-daemonset.yaml]: this `app` label shouldn't be here anymore now I think

Comment (rajatvig) [reply manifests/node-exporter-daemonset.yaml]: Nice catch. Fixed it and resolved conflicts.

**Comment by rajatvig** [comment]
Had to update the manifests other than those for node exporter as they seem out of date.

**Comment by brancz** [comment]
Lgtm but giving @lilic the last call.

**Comment by lilic** [comment]
Thanks!

**Pull request merged**